ZUGFeRD/InvoiceReferencedDocument.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
could you add the license on top of the file?

Reply:
I already added the license infos in a second commit, but the check still gives the error.
Do you know what else could cause the problem?